### ISSUE TITLE
SDK crashes on a payment

### ISSUE THREAD

**Issue by bracadabra** [ITECOMMPAY/paymentpage-sdk-android#6]
1. Clone the repo
2. Enter a project_id and secret
3. Launch the project and enter card details from the page https://developers.ecommpay.com/en/en_PP_TestCards.html
4. Press a button to pay

Actual result:
SDK crashes
```
java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=3, result=1, data=Intent { (has extras) }} to activity {com.ecommpay.test_integration_kotlin/com.paymentpagesdk.core.CoreActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'com.paymentpagesdk.core.entities.completefield.CompleteFieldEntity[] com.paymentpagesdk.core.entities.status.PaymentEntity.getCompleteFieldEntities()' on a null object reference
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4491)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4534)
        at android.app.ActivityThread.-wrap20(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1752)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6944)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'com.paymentpagesdk.core.entities.completefield.CompleteFieldEntity[] com.paymentpagesdk.core.entities.status.PaymentEntity.getCompleteFieldEntities()' on a null object reference
        at com.paymentpagesdk.core.components.presenters.result.DefaultResultPresenter.setupListViewWithData(DefaultResultPresenter.java:283)
        at com.paymentpagesdk.core.components.presenters.result.DefaultResultPresenter.present(DefaultResultPresenter.java:106)
        at com.paymentpagesdk.core.CoreActivity.showResultScreenErrorCode3DS(CoreActivity.java:1365)
        at com.paymentpagesdk.core.CoreActivity.onActivityResult(CoreActivity.java:1282)
        at android.app.Activity.dispatchActivityResult(Activity.java:7599)
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4487)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4534) 
        at android.app.ActivityThread.-wrap20(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1752) 
        at android.os.Handler.dispatchMessage(Handler.java:105) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6944) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374) 
```

**Comment by AlexeyKhrameev** [comment]
Thank you for report
We will fixed it at next release

**Comment by AlexeyKhrameev** [comment]
fixed at [1.14.2](https://github.com/ITECOMMPAY/paymentpage-sdk-android/releases/tag/1.14.2)
